Documentation/ceph-upgrade.md

```diff
@@ -529,9 +532,26 @@ are updated.
 
 ```console
 # kubectl --namespace rook-ceph get pod -o jsonpath='{range .items[*]}{range .spec.containers[*]}{.image}{"\n"}' -l 'app in (csi-rbdplugin,csi-rbdplugin-provisioner,csi-cephfsplugin,csi-cephfsplugin-provisioner)' | sort | uniq
-quay.io/cephcsi/cephcsi:v1.2.2
+quay.io/cephcsi/cephcsi:v2.0.1
 quay.io/k8scsi/csi-attacher:v1.2.0
 quay.io/k8scsi/csi-node-driver-registrar:v1.2.0
 quay.io/k8scsi/csi-provisioner:v1.4.0
 quay.io/k8scsi/csi-snapshotter:v1.2.2
+quay.io/k8scsi/csi-resizer:v0.4.0
 ```
+### Updating cephcsi from v2.0.0 to v2.0.1
+
+If you have a `v2.0.0` cluster, nodes with mounted Ceph persistent volumes will [hang after reboot](ceph-common-issues.md#node-hangs-after-reboot).
```

Comment:
This will only affect users of the v2.0.0 driver? I thought it was any previous version of ceph-csi.

Reply:
Yes it will affect all versions of csi